build.gradle

```diff
@@ -1,18 +1,22 @@
-group 'krakenApi'
-version '1.0-SNAPSHOT'
-
-apply plugin: 'java'
-
-sourceCompatibility = 1.8
-
-repositories {
-    mavenCentral()
-}
-
-dependencies {
-    compile 'com.fasterxml.jackson.core:jackson-databind:2.8.9'
-    compile 'org.apache.commons:commons-lang3:3.6'
-    compile 'org.apache.httpcomponents:httpclient:4.5.3'
-
-    testCompile group: 'junit', name: 'junit', version: '4.12'
-}
+plugins {
+    id 'java'
+    id "io.freefair.lombok" version "5.0.0-rc6"
+}
+
+group 'krakenApi'
+version '1.0-SNAPSHOT'
+
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.8.9'
```

Comment:
All deps should be "implementation".
Also should we upgrade all libs versions here too ?